libs/labelbox/src/labelbox/schema/project.py

```diff
@@ -710,6 +711,56 @@ def _connect_default_labeling_front_end(self, ontology_as_dict: dict):
             },
         )
 
+    def get_mal_prediction_imports(self) -> PaginatedCollection:
+        """Returns mal prediction import objects which are used in model-assisted labeling associated with the project.
+
+        Returns:
+            PaginatedCollection
+        """
+
+        id_param = "projectId"
+        query_str = """
+        query getModelAssistedLabelingPredictionImportsPyApi($%s: ID!) {
+            modelAssistedLabelingPredictionImports(skip: %%d, first: %%d, where: { projectId: $%s  }) { %s }}
+        """ % (
+            id_param,
+            id_param,
+            query.results_query_part(MALPredictionImport),
+        )
+
+        return PaginatedCollection(
+            self.client,
+            query_str,
+            {id_param: self.uid},
+            ["modelAssistedLabelingPredictionImports"],
+            MALPredictionImport,
+        )
+
+    def get_label_imports(self) -> PaginatedCollection:
```

Comment:
Note: Not going to add a delete path for regular label imports. This is because the mutation is internal and it can also put the data row in a bad state i.e. label with no annotations. We also already have SDk methods to delete and requeue labels so not needed.